mavros_msgs/msg/OpenDroneID/BasicID.msg

```diff
@@ -0,0 +1,30 @@
+# Remote ID message - Basic ID
+
+std_msgs/Header header
+
+uint8 id_type
+uint8 MAV_ODID_ID_TYPE_NONE = 0
+uint8 MAV_ODID_ID_TYPE_SERIAL_NUMBER = 1
+uint8 MAV_ODID_ID_TYPE_CAA_REGISTRATION_ID = 2
+uint8 MAV_ODID_ID_TYPE_UTM_ASSIGNED_UUID = 3
+uint8 MAV_ODID_ID_TYPE_SPECIFIC_SESSION_ID  = 4
```

Comment:
https://github.com/mavlink/mavros/blob/ros2/mavros_cog.py#L217

I'd prefer to remove unneded prefix - anyway it'll be namespaced into msg class.